### PR TITLE
Introduce grti and gti commands

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: aicoe/sesheta/chatbot.py [hunk 1]
```diff
@@ -121,6 +139,13 @@ async def process_user_text(thread_id: str, text: str) -> str:
 
         return f"I have told TektonCD to deliver `{intent[2]['tag']}` of repository `{intent[2]['repo_name']}`"
 
+    if intent[0] == "gti":
+        return ", ".join(_THOTH_INHABITANTS)
+
+    if intent[0] == "grti":
+        return f"⭐ In this Universe, based on relative position of planets and all the galaxies " \
+               f"I picked {random.choice(_THOTH_INHABITANTS)} ⭐"
```
Comment (harshad16) [inline aicoe/sesheta/chatbot.py:147]:
```suggestion
               f"I picked {hangouts_userid(random.choice(_THOTH_INHABITANTS))} ⭐"
```
with this the google chat would ping the user.

**Review by harshad16 — COMMENTED** [review]
Thanks this would be fun,
lgtm.

Just a minor comment/suggestion.

File: aicoe/sesheta/chatbot.py (hunk 1 above)
Comment (goern) [reply aicoe/sesheta/chatbot.py:147]: d'uh, I manually ported that!

**Pull request merged**